www/docs/en/dev/platform_plugin_versioning_ref/index.md

```diff
@@ -57,29 +59,76 @@ After running the above command, the **`package.json`** should contain something
 }
 ```
 
-The `--nosave` flag prevents adding and deleting specified platforms to the `package.json` file. To prevent saving a platform, issue the following command:
+The `--nosave` flag prevents adding and deleting the specified platform from the `package.json` file.
+
+Example:
 
 ```bash
 cordova platform add <platform[@<version>] | directory | git_url> --nosave
 ```
 
-Some Examples:
+Here are various ways to add platforms.
 
-* **`cordova platform add android`** 
+*Adding with Cordova resolved name:*
 
-  Retrieves the pinned version of `cordova-android` platform from npm, adds it to the project and updates the `package.json` file.
+Example:
 
-* **`cordova platform add android@7.1.4`**
+```bash
+cordova platform add android
+```
 
-  Retrieves the `cordova-android` platform version `7.1.4` from npm, adds it to the project and updates the `package.json` file.
+The avaialble Cordova Resolved names are:
 
-* **`cordova platform add https://github.com/apache/cordova-android.git`**
-  
-  **`cordova platform add https://github.com/apache/cordova-android`**
-  
-  **`cordova platform add github:apache/cordova-android`**
+* `android`
+* `electron`
+* `ios`
+* `browser`
+
+
+The `cordova-android` platform will be fetched from npm, added to the project and updates the `package.json` file.
```

Comment:
```suggestion
The latest version of the `cordova-android` platform will be fetched from npm, added to the project and the `package.json` file will be updated.
```

`updates` here and the other paragraphs sounds like strange English to me but I'm not a native speaker.